Gruntfile.js

```diff
@@ -1,28 +1,53 @@
 'use strict';
 
 module.exports = function(grunt) {
+  var fs = require('fs');
+  var path = require('path');
+  var isWindows = /^win/.test(process.platform);
 
   require('load-grunt-tasks')(grunt);
+  grunt.loadNpmTasks('grunt-contrib-concat');
 
+  // config
   var config = {
     libName: 'css-layout',
     distFolder: 'dist',
     srcFolder: 'src',
     testFolder: 'src/__tests__',
-    cTestFiles: 'src/__tests__/Layout-test.c src/Layout.c src/Layout-test-utils.c',
-    cTestOutput: 'c_test',
     javaLibFolder: 'src/java/lib',
     javaSource: 'src/java/tests/com/facebook/csslayout/*.java',
     javaTestFiles: 'org.junit.runner.JUnitCore com.facebook.csslayout.LayoutEngineTest com.facebook.csslayout.LayoutCachingTest com.facebook.csslayout.CSSNodeTest'
   };
 
+  // Create the dist folder if it doesn't exist. It is deleted by the 'clean' task.
```

Comment:
Its a side effect, but, since the dist folder can also be removed "manually" by someone that does the cleanup by hand, and the fact that the build fails if the dist folder doesn't exist I opted to make sure the folder exists.

I'm not sure whether grunt has some native task to create directories, which would probably be nicer than this.